From 9428bb9141fdc7d31c0e532f43e974ae2968eff6 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:26:08 -0400
Subject: [tools] perf/evsel: Add missing 'mmap2' from debug print

Message-id: <1408462094-14194-200-git-send-email-jolsa@redhat.com>
Patchwork-id: 88166
O-Subject: [PATCH RHEL7.1 BZ1131394 199/325] perf evsel: Add missing 'mmap2' from debug print
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 40d54ec2f77edc52340dcae236aaabe8c3cc3a07
Author: Adrian Hunter <adrian.hunter@intel.com>
Date: Fri Oct 18 15:28:58 2013 +0300

description
===========
The struct perf_event_attr now has a 'mmap2' member.  Add it to
perf_event_attr__fprintf().
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/evsel.c b/tools/perf/util/evsel.c
index bab6775..83a8be2 100644
--- a/tools/perf/util/evsel.c
+++ b/tools/perf/util/evsel.c
@@ -985,6 +985,7 @@ static size_t perf_event_attr__fprintf(struct perf_event_attr *attr, FILE *fp)
  ret += PRINT_ATTR2(exclude_host, exclude_guest);
  ret += PRINT_ATTR2N("excl.callchain_kern", exclude_callchain_kernel,
        "excl.callchain_user", exclude_callchain_user);
+ ret += PRINT_ATTR_U32(mmap2);
 
  ret += PRINT_ATTR_U32(wakeup_events);
  ret += PRINT_ATTR_U32(wakeup_watermark);
-- 
1.7.1